### DISCUSSION TITLE
入机一样

### DISCUSSION THREAD

**Discussion by shirekemi** [share121/inter-knot#5237] [委托]
宿舍有一个天天不洗澡的人😱，求求你洗澡吧呜呜呜呜
![Uploading mmexport1723351511336.jpg…]()

**Comment by miheihuyou** [comment]
实在不行，你隐晦的提醒一下吧

**Reply by shirekemi** [reply]
而且手机还外放，天天开麦打游戏到晚上 12 点

**Reply by miheihuyou** [reply]
我的妈呀，你们宿舍有宿舍长吗，或者你们一起讨论一个寝室规则呢

**Reply by shirekemi** [reply]
> 我的妈呀，你们宿舍有宿舍长吗，或者你们一起讨论一个寝室规则呢

没用，给他讲他也就装模作样洗一天，还不换衣服

**Reply by miheihuyou** [reply]
辅导员那里有帮忙做过调节吗？实在受不了换寝吧，我记得表白墙朋友圈可以找适合的室友一起组合找老师换寝

**Reply by shirekemi** [reply]
我们因为专业原因男生只有一个宿舍的人，老师也不好调，哎，刚开学也不想关系闹这么僵…
